Contributing.md

```diff
@@ -30,7 +30,7 @@ The general idea is to keep your 'master' branch in-sync with the 'upstream/mast
 
 ## Building JBoss Fuse Tooling
 
-To build _JBoss Fuse Tooling_ requires specific versions of Java (1.8+) and +Maven (3.0+). See this [link](https://github.com/jbosstools/jbosstools-fuse/blob/master/Build.md) for more information on how to setup, run and configure build.
+To build _JBoss Fuse Tooling_ requires specific versions of Java (1.8+) and +Maven (3.0+). See this [link](./Build.md) for more information on how to setup, run and configure build.
```

Comment:
in the example they are not using a "./Build.md" but directly "Build.md".
Do you know if there is a difference?

Reply:
no idea. I did not look at the examples. I tested the link with ./filename and that seems to work fine.